pandas/core/frame.py

```diff
@@ -910,17 +911,35 @@ def itertuples(self, index=True, name="Pandas"):
 
         Examples
         --------
-        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [0.1, 0.2]},
-        ...                   index=['a', 'b'])
+        >>> df = pd.DataFrame({'num_legs': [4,2], 'num_wings': [0,2]},
```

Comment:
Can you add spaces after the commas? You can use `flake8 --doctests frame.py`. It will generate many messages from other docstrings, but you can check the line numbers of yours, and make sure there is nothing wrong between them.

Reply:
Sorry totally forgot about that second test. Fixed the whitespacing between the commas.